### DISCUSSION TITLE
Mobile not formatting correctly

### DISCUSSION THREAD

**Discussion by Hankenoj** [alshedivat/al-folio#1708] [Q&A] [answered]
Hello my site (henryelgie.com) works great on desktop applications but when i open it in mobile no image shows from my profile picture. No menu is avalible on the top right corner. Any tips on getting this to work. I am using nginx webserver vultr hosting and jekyll. 

Attached is my config.yml

**Comment by Hankenoj** [comment]
Update to this discussion for some reasion when i open mysite and click blog projects wtv it looks for /al-folio/blog but when i create a al-folio folder and place blog in that folder it looks just for domain.com/blog

**Comment by Hankenoj** [comment]
Another update: I got it to work by duplicateing all of my blog, projects, assets, etc folders into a al-folio folder and then leaveing another copy of those folders in the root of the site. This was really werid any way to fix it?

**Comment by george-gca** [comment, marked as answer]
Your `_config.yml` file was not attached, but from what you told, you should leave this field [here](https://github.com/alshedivat/al-folio/blob/5a96b75479fbb562823b59ee9821d999233faac1/_config.yml#L22C1-L22C59) empty.

**Reply by Hankenoj** [reply]
yup that was the issue. I also found that google cached older versions of the site so it didnt update instantly. The site works great now.
